src/sentry_options.c

```diff
@@ -24,6 +24,7 @@ sentry_options_new(void)
     opts->debug = debug && sentry__string_eq(debug, "1");
     sentry_logger_t logger = { sentry__logger_defaultlogger, NULL };
     opts->logger = logger;
+    opts->bgworker_name = sentry__string_clone("Sentry BgWorker");
```

Comment:
```suggestion
    opts->bgworker_name = sentry__string_clone("sentry-transport");
```

Reply:
Need to test it, according to @xamix, it could be only 15 chars (without null-terminator)

Reply:
Yes it will fail on Linux and returning error code `E_RANGE`.

Reply:
`sentry-http` is 11 chars which should work and looks a bit nicer